ledger/error/src/main/scala/com/daml/error/definitions/groups/UserManagementServiceErrors.scala

```diff
@@ -15,6 +15,74 @@ import com.daml.error.{
 
 object UserManagementServiceErrors extends AdminServices.UserManagementServiceErrorGroup {
```

Comment:
Shouldn't we rename this class and file to `UserManagementServiceErrorGroup`?

Reply:
Ok